spec/services/atmosphere/cloud/satisfy_appliance_spec.rb

```diff
@@ -17,6 +17,73 @@
   let!(:openstack) { create(:openstack_with_flavors, funds: [fund]) }
   let!(:tmpl_of_shareable_at) { create(:virtual_machine_template, appliance_type: shareable_appl_type, tenants: [openstack])}
 
+  context '#assign fund' do
+    let!(:t) { create(:openstack_with_flavors, active: true, funds: [fund]) }
+    let!(:vmt) { create(:virtual_machine_template, tenants: [t]) }
+    let!(:appliance_type) { create(:appliance_type, preference_cpu: 0, \
+      preference_disk: 0, preference_memory: 0, virtual_machine_templates: [vmt]) }
```

Comment:
Line is too long. [83/80]<br>Align the elements of a hash literal if they span more than one line.<br>Expression at 24, 83 should be on its own line.